src/main/java/Deadline.java

```diff
@@ -0,0 +1,10 @@
+public class Deadline extends Task{
+    protected String by;
+    public Deadline(String description, String by){
+        super(description);
+        this.by = by;
+    }
+    public String toString(){
+        return "[D]" + super.toString() + "by: " + by + ")";
+    }
```

Comment:
It is better to follow the coding standard where there is a space between the class name and the {. The same goes for methods.